packages/lab/src/calendar/internal/CalendarDay.tsx

```diff
@@ -52,7 +55,7 @@ export const CalendarDay = forwardRef<HTMLButtonElement, CalendarDayProps>(
 
     const dayRef = useRef<HTMLButtonElement>(null);
     const buttonRef = useForkRef(ref, dayRef);
-    const { status, dayProps, unselectableReason, highlightedReason, locale } =
+    const { status, dayProps, unselectableReason, highlightedReason } =
```

Comment:
Unselectable and disabled styles are potentially wrong (shown as single horizontal stroke). One should be a cross and one should be no strikethrough?

Reply:
will follow up on this item in a  new PR